packages/amplify-category-api/src/provider-utils/awscloudformation/utils/rds-secrets/database-secrets.ts

```diff
@@ -66,21 +69,19 @@ export const getExistingConnectionSecretNames = async (context: $TSContext, conf
     }, {});
 
     if (existingSecrets && (Object.keys(existingSecrets)?.length === secretNames?.length)) {
-      existingSecrets.database = config.database;
-      existingSecrets.host = config.host;
-      existingSecrets.port = config.port;
+      existingSecrets.database = database;
```

Comment:
Have you tested this out with the lambda yet? I suspect this will break RDS connectivity as it won't know where the host or port are

Reply:
There's one change that's needed which is that the lambda should read these from SSM directly, that I need to make.

Reply:
Made the change